rdm/init_files/translate.py

```diff
@@ -0,0 +1,24 @@
+from rdm.util import write_yaml
```

Comment:
Why is `translate.py` present in the `init_files` directory?  It seems like we wouldn't want any python scripts to be in `init_files`, since they will be copied into projects when users run `rdm init`.

Reply:
Fixed.

Reply:
Fixed.